content/collections/reference/metadata.md

```diff
@@ -17,174 +17,187 @@ As an example, see the [complete metadata file](https://github.com/OpenTermsArch
     name="id"
     type="string"
     description="Unique identifier derived from name (acronyms, dash-separated)."
-    example="`pga`"
+    example="pga"
```

Comment:
Why do we take `pga` as an example, which is a third-party that we don't control, instead of `demo` that is here to that end?